WordPress-Core/ruleset.xml

```diff
@@ -356,6 +356,13 @@
 	<!-- Class opening braces should be on the same line as the statement. -->
 	<rule ref="Generic.Classes.OpeningBraceSameLine"/>
 
+	<!-- References to self in a class should be lower-case and not have extraneous spaces,
+		 per implicit conventions in the core codebase; the NotUsed code refers to using the
+		 fully-qualified class name instead of self, for which there are instances in core. -->
+	<rule ref="Squiz.Classes.SelfMemberReference"/>
+	<rule ref="Squiz.Classes.SelfMemberReference.NotUsed">
+		<severity>0</severity>
```

Comment:
@GaryJones The parts of the sniff which have been enabled for `Core` deal with whitespace only, which is quite clear-cut.

Whether or not to use `self` vs a `\Fully\Qualified\Classname` is something else altogether.

So independently of whether Core currently complies with it's own rules (in which neither of the above are covered), I do believe the `NotUsed` code is better placed in `Extra` than in `Core` as it is opinionated/best practice.